packages/eth2.0-types/src/validator.ts

```diff
@@ -2,13 +2,13 @@
  * @module types
  */
 
-import {BLSPubkey, bytes48, number64, Shard, Slot, uint64} from "./primitive";
+import {BLSPubkey, Shard, Slot, uint64, ValidatorIndex} from "./primitive";
 
 export interface ValidatorDuty {
   // The validator's public key, uniquely identifying them
   validatorPubkey: BLSPubkey;
   // The index of the validator in the committee
-  committeeIndex: number64;
+  committeeIndex: ValidatorIndex;
```

Comment:
I think this is a bit misleading. as index in committee is different than ValidatorIndex and it will never be uint or something